plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -379,7 +384,12 @@ else if (identity.getId() == TRINO_MERGE_PARTITION_DATA) {
                 session,
                 split.getFileFormat(),
                 table.getStorageProperties(),
-                maxOpenPartitions);
+                maxOpenPartitions,
+                table.getSortOrder(),
+                DataSize.of(100, DataSize.Unit.MEGABYTE),
```

Comment:
left-over from testing ? 

I assume we would need here `SortingFileWriterConfig`

Reply:
🤦 yeah thanks for catching that. I hardcoded these before I had the config wired up.